_posts/2018-10-29-bazel-0.19.0.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: posts
+title: "Bazel 0.19"
+authors:
+  - katre
+---
+
+The Bazel team is happy to announce a new version of Bazel, [Bazel 0.19](https://github.com/bazelbuild/bazel/releases/tag/0.19.0).
+
+This document lists the major changes since Bazel 0.18.
+
+## General changes
+
+* The [`--incompatible_expand_directories`](https://docs.bazel.build/versions/master/skylark/backward-compatibility.html#expand-directories-in-args) flag automatically expands directories in command lines. See the design document.
```

Comment:
where is the link to the design document?

Reply:
Does this need to be fixed before we can announce the release?

Reply:
Indeed, links from the original document are lost (https://docs.google.com/document/d/1wDvulLlj4NAlPZamdlEVFORks3YXJonCjyuQMUQEmB0/edit)

Reply:
Fixed.